build_scripts/build_ffmpeg.sh

```diff
@@ -17,6 +17,14 @@
 # For a snapshot of the code, see the README.rst
 if [ ${WITH_FFMPEG} -gt 0 ]; then
     pushd third_party/FFmpeg
+    echo "--- a/libavcodec/exr.c
```

Comment:
how about using patch files like we do with other repos?

Reply:
We would need to include this in the source code we publish, so it complicates the flow a bit (we would need to host multiple variants of the same version + patches). Now it is a part of the build command so it should be fine.

Reply:
Makes sense